files/setup-08-tinywww.sh

```diff
@@ -13,6 +13,11 @@ VEBA_CONFIG_FILE=/root/config/veba-config.json
 TINYWWW_TEMPLATE=/root/config/tinywww/templates/tinywww-template.yaml
 TINYWWW_CONFIG=/root/config/tinywww/tinywww.yaml
 
+# Basic Auth for TinyWWW endpoints
+kubectl -n vmware-system create secret generic basic-auth \
```

Comment:
remind me, why and where is this needed now?

Reply:
To access any of the endpoints served by TinyWWW - /events, /bootstrap, etc.

Basically, we don't want someone to be able to view events or debug logs w/o some sort of auth

Reply:
ACK. And this is not handled by an Ingress? I thought we had those endpoints exposed through ingress which also has auth?

Reply:
No, we've always had it like this